.github/workflows/images.yml

```diff
@@ -26,18 +26,16 @@ jobs:
       images: ${{ steps.images-matrix.outputs.images }}
     steps:
       - uses: actions/checkout@v4
+
+      - name: Install Bowtie
+        uses: ./
+        with:
+          installation-attempts: "5"
+          installation-wait-seconds: "10"
+
       - id: images-matrix
         run: |
-          python3 -c '
-          from pathlib import Path
-          import json
-          paths = [
-              str(path.name)
-              for path in Path("implementations").iterdir()
-              if path.is_dir()
-          ]
-          print(f"images={json.dumps(paths)}")
-          ' >> $GITHUB_OUTPUT
+          echo "images=$(bowtie filter-implementations | jq -R -s -c 'split("\n")[:-1]')" >> $GITHUB_OUTPUT
```

Comment:
I added `bowtie filter-implementations --format json`, so there's no need for ad hoc parsing here.

We don't apparently do the same "magic" when stdout isn't a tty (meaning making format JSON be the default when piping), though we could always change that later.

But yeah don't do text parsing.